cms/check_report_used_space

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python
+# Copyright European Organization for Nuclear Research (CERN) 2013
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# You may not use this file except in compliance with the License.
+# You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+#
+# Authors:
+# - Donata Mielaikaite, <donata.mielaikaite@cern.ch>, 2020
+# - Fernando Garzon, <oscar.fernando.garzon.miguez.ch> 2020
+
+'''
+Probe to check used space.
+'''
+
+import datetime
+import unicodedata
+import sys
+import traceback
+from rucio.core import monitor
+from rucio.core.rse import list_rses, get_rse_usage, list_rse_attributes
+from rucio.db.sqla import models
+from rucio.db.sqla.session import BASE, get_session
+
+# Exit statuses
+OK, WARNING, CRITICAL, UNKNOWN = 0, 1, 2, 3
+
+if BASE.metadata.schema:
+    schema = BASE.metadata.schema + '.'
+
+if __name__ == "__main__":
+    try:
+        status = OK
+        session = get_session()
+        for rse in list_rses():
+            limits = get_rse_usage(rse['id'])
+            attributes = list_rse_attributes(rse['id'])
+            for usage in limits:
+                source = usage['source']
+                country = attributes.get('country', 'UNKNOWN')
+                query = session.query(models.RSE.rse_type).filter(models.RSE.id == rse['id'])
+                result = query.one()
+                rse_type = result[0]
```

Comment:
I have a preference for using `rucio.core.rse.get_rse`, which allows you to get rid of the database-related imports. If you disagree, at least use `.scalar()` and combine everything in a single statement.